app/models/organization.rb

```diff
@@ -293,6 +314,27 @@ def update_associations
     calculated_children.each { |o| o.update_attributes(updated_at: Time.current, skip_update: true) }
   end
 
+  def search_location
+    locations.first
+  end
+
+  def regional?
+    paid_feature_slugs.include?("regional_bike_counts")
+  end
+
+  def coordinates_set?
+    location_latitude.present? && location_longitude.present?
+  end
+
+  def regional_suborganizations
+    return Organization.none unless regional? && coordinates_set?
+
+    address = search_location&.address
+    return Organization.none unless address.present?
+
+    Organization.near(address, search_radius).where.not(id: id)
```

Comment:
Why use address here and not `[latitude, longitude]`? I believe Geocoder will just geocode that location and query based on the lat, long anyway...

Reply:
Makes sense — I think that'll save an API call